config/application.yml.example

```diff
@@ -1,2 +1,3 @@
+HOST: 'http://localhost:3000'
```

Comment:
Are all places that use `HOST` going to support the protocol being included or should the host variable omit it?

Reply:
Good question. I guess I'll just remove it.